src/main/java/ua/softserveinc/tc/util/DateUtil.java

```diff
@@ -41,6 +43,10 @@ public static Date toDate(String date) {
         }
     }
 
+    public static Date toDate(LocalDate date) {
```

Comment:
Please try think again)))
This method is always return time as 00:00:00. Don't argue any more, just test it!